eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -56,9 +56,9 @@
 
     private CodeCheckerContext() {
         PerServerActionRunner<ThriftCommunicationInterface> actionRunner;
-        actionRunner = new PerServerSimpleActionRunner<ThriftCommunicationInterface>(new
-                ThriftTransportFactory(), (new CodecheckerActionInitializer()).initialize(new
-                ActionImplementationRegistry<ThriftCommunicationInterface>()));
+        actionRunner = new PerServerSimpleActionRunner<ThriftCommunicationInterface>(new ThriftTransportFactory(),
```

Comment:
it's usually a good idea to don't change the formatting (or do so in a separate commit), as it can make merging more complicated. Especially for files totally unrelated to the commit message.